api/applications/applicationRouter.js

```diff
@@ -347,4 +347,59 @@ router.put(
   }
 );
 
+/**
```

Comment:
Using Swagger to update your API's Documentation is strong signal to me and the WORLD that you care about organization and the quality of the code you write.

Reply:
Seconding this - seeing updated Swagger documentation is always a good sign of your competency and dedication!